win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1123,8 +1137,14 @@ BITMAP_LoadImageW(
         pbmi = LockResource(hgRsrc);
         if(!pbmi)
             return NULL;
+        ResSize = SizeofResource(hinst, hrsrc);
     }
 
+    if (pbmi->bmiHeader.biSizeImage + pbmi->bmiHeader.biSize + 12 != ResSize &&
+        pbmi->bmiHeader.biCompression == BI_BITFIELDS &&
+        pbmi->bmiHeader.biBitCount == 32)
+        ERR("Possibly bad resource size provided\n");
```

Comment:
Here `ERR()`, below `WARN()`: are both needed? Why not same level?

Reply:
I have swapped the ERR() and the WARN(). Now. In the first case, this is based on my limited testing and may not actually be an error, but is something that I believe should be watched and maybe more conditions should be placed within it conditional statement. For the one below, it is a definite failing of GNU's GCC windres.exe to handle this situation correctly. Therefore, I used ERR() here. If this is fixed, then the whole section can be removed.